### PR TITLE
Implement Signal toolUpdate/Down

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/Windows/Construction/Construction.h [hunk 1]
```diff
@@ -52,6 +52,12 @@ namespace OpenLoco::Ui::Windows::Construction
     static loco_global<uint16_t, 0x01135FE4> _lastSelectedMods;
     static loco_global<uint16_t, 0x01135FFE> _word_1135FFE;
     static loco_global<int16_t, 0x01136000> _word_1136000;
+    static loco_global<uint16_t, 0x01136002> _signalGhostSides;
+    static loco_global<Map::Pos3, 0x01136004> _signalGhostPos;
+    static loco_global<uint8_t, 0x0113600D> _signalGhostRotation;
+    static loco_global<uint8_t, 0x0113600E> _signalGhostTrackId;
+    static loco_global<uint8_t, 0x0113600F> _signalGhostTileIndex;
+    static loco_global<uint16_t, 0x0113600A> _signalGhostTrackObjId;
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Windows/Construction/Construction.h:60]: Worth moving these into SignalTab.cpp? I'd like to avoid loco globals in headers if we can.

Comment (duncanspumpkin) [reply src/OpenLoco/Windows/Construction/Construction.h:65]: They will be used by the common ghost removal code unfortunately so will need to stay here.

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/Construction/Construction.h:65]: Okay, fine for now, then. At some point, these variables are going to have to be integrated in one location, though.

Comment (duncanspumpkin) [reply src/OpenLoco/Windows/Construction/Construction.h:65]: I've got an idea for how to move them but will wait until ive done the common ghost removal code.

**Review by AaronVanGeffen — COMMENTED** [review]
Signal ghosts aren't removed properly:

Screenshot (1)

**Comment by duncanspumpkin** [comment]
Woops wrote to the wrong global address and as ghost removal is still original code it couldn't find the signal to remove.

**Review by AaronVanGeffen — APPROVED** [review]
Now works as expected

**Pull request merged**